pkg/nsx/services/securitypolicy/expand.go

```diff
@@ -111,7 +111,7 @@ func (service *SecurityPolicyService) expandRuleByService(obj *v1alpha1.Security
 		ruleIPSetGroup := service.buildRuleIPSetGroup(obj, rule, nsxRule, portAddress.IPs, ruleIdx)
 		groupPath := fmt.Sprintf(
 			"/infra/domains/%s/groups/%s",
-			getDomain(service),
```

Comment:
we had better to keep getDomain function or put it as common function, although it's short function, but it's more meaningful compared to use "service.NSXConfig.Cluster".